### DISCUSSION TITLE
Porting to Blazor

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [fel88/Dendrite#9] [Ideas]
Currently the UI is based on .NET Framework WinForm

It could draw attraction and attention if the UI is replaced with Blazor or Migrating .NET Framework to .NET6 WinForm

**Comment by fel88** [comment]
Actually, I thought about porting UI to OpenGL (OpenTK) or Skia

**Comment by GeorgeS2019** [comment]
Go for Blazor using SkiaSharp ==> https://www.nuget.org/packages/SkiaSharp.Views.Blazor

https://github.com/mattleibow/SkiaSharpBlazorWebAssembly

https://github.com/mono/SkiaSharp/issues?q=+Blazor

I know you prefer OpenGL using OpenTK, Skia gives u better Future attraction

https://github.com/danroth27/BlazorSkiaSharp

https://github.com/max1287/SkiaSharpBlazorApp

https://github.com/JensKrumsieck/OxyPlot.SkiaSharp.Blazor
